app/helpers/number_helper.rb

```diff
@@ -0,0 +1,82 @@
+module NumberHelper
+  def number_to_human_size(number, *args)
+    options = args.extract_options!.reverse_merge(:significant => false, :precision => 1)
+    NumberHelper.handling_negatives(number) do |number|
+      super(number, options)
```

Comment:
How does this work?  Does AR automatically load its own NumberHelper and load this as an extension?  Or is the fact that it's mixed into ApplicationHelper what does it?